1-exercises/A-reading-list/script.js

```diff
@@ -1,9 +1,29 @@
 function readingList(books) {
-  // Write your code here...
+  let contentEle = document.querySelector("#content");
```

Comment:
You could consider using `const` instead of `let`, it's more optimal for variables which don't change.